onnxruntime/core/providers/webnn/builders/model_builder.h

```diff
@@ -22,9 +22,12 @@ class IOpBuilder;
 class ModelBuilder {
  public:
   ModelBuilder(const GraphViewer& graph_viewer, const logging::Logger& logger,
-               const emscripten::val& context, const emscripten::val& builder,
-               const DataLayout preferred_layout, const WebnnDeviceType wnn_device_type);
-  ~ModelBuilder() = default;
+               const emscripten::val& context, const DataLayout preferred_layout,
+               const WebnnDeviceType wnn_device_type);
+  ~ModelBuilder() {
+    // Explicitly release the WebNN builder to free memory.
+    wnn_builder_ = emscripten::val::undefined();
```

Comment:
Given builder is not usable after graph built, should we release it right after calling `build()`?

Reply:
Thanks, good point, done!